certbot/certbot/_internal/auth_handler.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 import logging
 import time
-from typing import Dict
+from typing import Dict, Sequence
```

Comment:
Just a general question since you seem to prefer to keep the `typing` imports each on its own line? Also, this is not sorted.

Reply:
Yes, my IDE and me do not agree on the import formatting ;)